contrib/ckan/docker-entrypoint.d/openafrica_ckan.sh

```diff
@@ -16,6 +16,13 @@ fi
 ckan config-tool ${CKAN_INI} -s app:main -e ckan.auth.create_user_via_web=true
 ckan config-tool ${CKAN_INI} -s app:main -e ckan.auth.public_user_details=true
 
+#Auth settings, limit actions to sysadmin users
+ckan config-tool ${CKAN_INI} -s app:main -e ckan.auth.user_create_organizations=false
+ckan config-tool ${CKAN_INI} -s app:main -e ckan.auth.user_create_groups=false
+ckan config-tool ${CKAN_INI} -s app:main -e ckan.auth.create_dataset_if_not_in_organization=false
```

Comment:
From the [docs](https://docs.ckan.org/en/2.10/maintaining/configuration.html#ckan-auth-create-dataset-if-not-in-organization), `create_dataset_if_not_in_organization=True` should be meaningless if `create_unowned_dataset` is `False` (which should be by [default](https://docs.ckan.org/en/2.10/maintaining/configuration.html#ckan-auth-create-unowned-dataset)). Does this change anything?

Reply:
You/docs is correct.